Formula/kyma-cli.rb

```diff
@@ -2,9 +2,10 @@ class KymaCli < Formula
   desc "Kyma command-line interface"
   homepage "https://kyma-project.io"
   url "https://github.com/kyma-project/cli/archive/1.24.7.tar.gz"
-  sha256 "c2465d50f0f2c082fc9bc73ccfac4be824cc826980ab263b8b4ce00dcdcc5b9f"
+  sha256 "61dcec274376227c8130b2fd6106c5231f878d949f85972182dadacc0396e460"
```

Comment:
I am not sure where did you get this checksum. I just checked the `checksums.txt` in the kyma cli release 1.24.7: https://github.com/kyma-project/cli/releases/tag/1.24.7 here the checksum for darwin is different `6cf1e17fb2856adc6d6d18b4e111ca5b495a4722bc8a2750995c930332941b1b`.

Reply:
This is `1.24.7.tar.gz`, not `kyma_Darwin_x86_64.tar.gz`.

We do not use pre-built binaries in homebrew-core.

Reply:
thanks 👍